pages/common/fsck.md

```diff
@@ -0,0 +1,15 @@
+# fsck
+
+> Check the integrity of a filesystem or repair it.  The filesystem should be unmounted at the time the command is run.
+
+- Check filesystem /dev/sdX, reporting any damaged blocks:
```

Comment:
Perhaps a concrete example (e.g. /dev/sda) would be less prone to confusing newbies (is X meant to be a letter? a digit? uppercase?). But just a suggestion, feel free to disagree.